### DISCUSSION TITLE
Proposal: Invert fa-eye regular so the iris is dark again

### DISCUSSION THREAD

**Discussion by thethomic** [FortAwesome/Font-Awesome#21270] [Feature Requests]
hey there!
the new fa-eye looks good in "solid" but is inverted in "regular", that way it looks like it was drugged and somehow looking up left ;)
https://fontawesome.com/icons/eye?s=regular
![image](https://user-images.githubusercontent.com/49404031/154223867-6b06ab92-4d97-47ef-822c-9e2aa02f5f5c.png)

I propose to leave the iris dark, as it was in fa-5
![image](https://user-images.githubusercontent.com/49404031/154279914-66b24864-fbd8-4e72-815c-36176c0e6357.png)

best regards and keep up the good work!
thomic

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

Let's ask @sensibleworld

**Comment by tomours** [comment]
We are switching our app to V6, and we also found that this new eye is really awful.
We were going to upload the old V5 eye into our kits and change all our far to fak.
Can you tell us if you intend to rollback the eye style ? (so we won't have to change 2 times our icons)

Except that eye change, thanks for all you're doing, we really appreciate these icons ! :+1:
